### DISCUSSION TITLE
Welcome to art-point Discussions!

### DISCUSSION THREAD

**Discussion by vk-19** [IRFVIN/art-point#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by genericSpecimen** [comment]
I'm genericSpecimen and I'm helping with the development of Art Point.

**Comment by vk-19** [comment]
This is artpoint
